libskutils/include/skutils/utils.h

```diff
@@ -198,6 +198,49 @@ extern std::string nanoseconds_2_lifetime_str( uint64_t ns, bool isColored = fal
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+extern bool convert_json_string_value_to_boolean( const std::string r );
```

Comment:
It is better to use const reference here to avoid copying.

Reply:
Fixed